WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsRelatedPostsView.swift

```diff
@@ -0,0 +1,143 @@
+            }
+        }
+    }
+
+    private func save(field: String, value: Any) {
+        WPAnalytics.trackSettingsChange("related_posts", fieldName: field, value: value)
+        BlogService(coreDataStack: ContextManager.shared).updateSettings(for: blog, success: nil, failure: { _ in
+            SVProgressHUD.showDismissibleError(withStatus: Strings.saveFailed)
+        })
+    }
+}
+
+private struct PreviewViewModel: Identifiable {
+    let id = UUID()
+    let title: String
+    let details: String
+    let imageName: String
+
+    static let previews: [PreviewViewModel] = [
+        PreviewViewModel(
+            title: NSLocalizedString("relatedPostsSettings.preview1.title", value: "Big iPhone/iPad Update Now Available", comment: "Text for related post cell preview"),
+            details: NSLocalizedString("relatedPostsSettings.preview1.details", value: "in \"Mobile\"", comment: "Text for related post cell preview"),
+            imageName: "relatedPostsPreview1"
+        ),
+        PreviewViewModel(
+            title: NSLocalizedString("relatedPostsSettings.preview2.title", value: "The WordPress for Android App Gets a Big Facelift", comment: "Text for related post cell preview"),
+            details: NSLocalizedString("relatedPostsSettings.preview2.details", value: "in \"Apps\"", comment: "Text for related post cell preview"),
+            imageName: "relatedPostsPreview2"
+        ),
+        PreviewViewModel(
+            title: NSLocalizedString("relatedPostsSettings.preview3.title", value: "Upgrade Focus: VideoPress For Weddings", comment: "Text for related post cell preview"),
+            details: NSLocalizedString("relatedPostsSettings.preview3.details", value: "in \"Upgrade\"", comment: "Text for related post cell preview"),
+            imageName: "relatedPostsPreview3"
+        )
+    ]
+}
+
+private extension RelatedPostsSettingsView {
+    enum Strings {
+        static let title = NSLocalizedString("relatedPostsSettings.title", value: "Related Posts", comment: "Title for screen that allows configuration of your blog/site related posts settings.")
+        static let showRelatedPosts = NSLocalizedString("relatedPostsSettings.showRelatedPosts", value: "Show Related Posts", comment: "Label for configuration switch to enable/disable related posts")
+        static let showHeader = NSLocalizedString("relatedPostsSettings.showHeader", value: "Show Header", comment: "Label for configuration switch to show/hide the header for the related posts section")
+        static let showThumbnail = NSLocalizedString("relatedPostsSettings.showThumbnail", value: "Show Images", comment: "Label for configuration switch to show/hide images thumbnail for the related posts")
+        static let optionsFooter = NSLocalizedString("relatedPostsSettings.optionsFooter", value: "Related Posts displays relevant content from your site below your posts", comment: "Information of what related post are and how they are presented")
```

Comment:
👋🏻 @kean I noticed [a translation inquiry](https://wordpress.slack.com/archives/C02RQC4LY/p1689091019847819) in the WordPress Slack regarding Related Posts. Specifically combining plural "Related Posts" with "displays," i.e Related Posts **displays**. Do you have thoughts on whether this string should be adjusted?

Reply:
Hey, @dcalhoun. I don't think I'm on this Slack. I would appreciate it if you could reply.

The value was copied from the original version. I simply added the proper key.

```swift
return NSLocalizedString(@"Related Posts displays relevant content from your site below your posts", @"Information of what related post are and how they are presented");;
```

I think it means "Related Posts (section) displays" or something like that, but "display" sounds more natural in this context.

Reply:
Please, disregard my comment. I created the account and replied directly in the thread.